src/Utilities/Compiler/Options/Unit.cs

```diff
@@ -45,7 +45,9 @@ internal struct Unit : IEquatable<Unit>
         /// <param name="first">The first <see cref="Unit"/> value to compare.</param>
         /// <param name="second">The second <see cref="Unit"/> value to compare.</param>
         /// <returns>Because <see cref="Unit"/> has a single value, this always returns <c>true</c>.</returns>
-        [SuppressMessage("Microsoft.Usage", "CA1801:ReviewUnusedParameters", MessageId = "first", Justification = "Parameter required for operator overloading."), SuppressMessage("Microsoft.Usage", "CA1801:ReviewUnusedParameters", MessageId = "second", Justification = "Parameter required for operator overloading.")]
+        [SuppressMessage("Microsoft.Usage", "CA1801:ReviewUnusedParameters", MessageId = "first", Justification = "Parameter required for operator overloading.")]
+        [SuppressMessage("Microsoft.Usage", "CA1801:ReviewUnusedParameters", MessageId = "second", Justification = "Parameter required for operator overloading.")]
+        [SuppressMessage("Style", "IDE0060:Remove unused parameter", Justification = "Parameter required for operator overloading.")]
```

Comment:
It'd be nice to have `CA1801` and `IDE0060` mutually exclusive.

Reply:
Yep, need a suppressor.